web/installer

```diff
@@ -11,7 +11,7 @@
  * file that was distributed with this source code.
  */
 
-process($argv);
+process( is_array($argv) ? $argv : array() );
```

Comment:
Could you give me a hint on this? Only thing I've found on Google regarding this is there is no standard. Should I just use a multi-line if else blocks?